openmsupply/conforma-server
# #18 Extend action to remove org join to all users

Based in: #874 (I think it was the one used for latest build...)
Fixes https://github.com/openmsupply/conforma-templates/issues/18

### Testing
- Load [disableOrg-3 template](https://github.com/openmsupply/conforma-server/files/9760354/disableOrg-3.zip) in latest snap shot. Make sure is set as Available.
- Try removing one organisation. (ex: 13) - Log in as `manager` and use the template (Top Menu => Gerência => Desativar Entidade)
- See how it's removed by running this Graphql query **before** and **after**

### Changes

1. Allow passing `username` and/or `orgName` instead of ides
2. Enforces that some orgId or orgName was passed - it cannot be null
3. When no userId or username is passed (`undefined`) will interpret as remove ALL used connected to org
4. Keeps same logic to remove permissionJoins - for the pair `{userId, orgId}` or for all users connected to org (when no userId is defined).

~~TODO: Update documentation~~
